### DISCUSSION TITLE
New Icon

### DISCUSSION THREAD

**Discussion by diegopvlk** [andyholmes/valent#541] [General]
Hey, I made an icon for Valent that I been using for a while because I find the default to be a little basic :P
If you liked and are interested, I can make a new PR (or I can send you the .svg, whatever you prefer). This is how it looks:

![image](https://github.com/andyholmes/valent/assets/25941156/2ebacb38-dc6f-4342-9056-309d68a6aa9c)

**Comment by andyholmes** [comment]
That looks great!

If you could open a PR with a "devel" icon included, I would totally accept it :slightly_smiling_face:. Is there a "source" SVG as well? If so it would be nice to include that in the same directory.

I've just pushed a commit to add an Artists section, so you can add your name [here](https://github.com/andyholmes/valent/blob/main/src/libvalent/ui/valent-application-credits.h#L11), too.

**Comment by diegopvlk** [comment]
Done. I don't really need the credits, I'll leave up to you if want to add my name or not =) cheers!
